api/customers/customersRouter.js

```diff
@@ -95,12 +96,21 @@ router.put('/:id', async (req, res) => {
       const newFav = [...oldFav, req.body.favorite_groomers];
       let newBody = req.body;
       newBody.favorite_groomers = newFav;
+      console.log(newBody);
```

Comment:
Could probably leave this console.log out but since it's BE, it'll be fine. Nice work!